src/main/java/com/google/sps/data/Section.java

```diff
@@ -67,4 +67,9 @@ public boolean overlaps(Section other){
         //TODO: This
         return false;
     }
-}
\ No newline at end of file
+
+    // Getter for meetingTimes
+    public List<TimeRange> getMeetingTimes() {
```

Comment:
We might run into a merge conflict here, as I added the same function! But depending on which pull request gets merged first, I'll edit mine so that it shouldn't be too gnarly.

Reply:
Ah I see. Thats fine. I'll keep that in mind when resolving merge conflicts.